src/filters/filters.less

```diff
@@ -28,3 +28,70 @@
     }
   }
 }
+
+pf-filter-panel {
+  .dropdown > .dropdown-menu, .input-group-btn > .dropdown-menu {
+    margin-top: 5px;
+  }
+  input {
+    &:-moz-placeholder            { font-style: italic; padding-left: 10px; } // Firefox 4-18
+    &::-moz-placeholder           { font-style: italic; padding-left: 10px; } // Firefox 19+
+    &:-ms-input-placeholder       { font-style: italic; padding-left: 10px; } // Internet Explorer 10+
+    &::-webkit-input-placeholder  { font-style: italic; padding-left: 10px;} // Safari and Chrome
+  }
+  .filter-panel-container {
+    padding: 10px;
+    .keyword-filter {
+      margin-bottom: 10px;
+    }
+    ul {
+      list-style-type: none;
+      padding-left: 16px;
+    }
+    .category {
+      font-size: 15px;
+      font-weight: 400;
+
+      .category-option-label {
+        vertical-align: text-bottom;
+        font-size: 12px;
+      }
+    }
+  }
+  .list-inline > li {
+    padding-right: 0px;
+    @media(max-width: @screen-md-min) {
+      margin-bottom: 3px;
+    }
```

Comment:
Ok, but not sure what you are suggesting?  Do more to resolve the responsiveness, remove the bootstrap variables.less usage, perhaps add our own @pf-screen-md-min, etc..?